LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Apache licence relates to software.

For the docs we should you use [CC-BY NC-SA](https://creativecommons.org/licenses/by-nc-sa/3.0/) see:
https://github.com/home-assistant/home-assistant.io/blob/current/LICENSE.md
https://github.com/home-assistant/companion.home-assistant/blob/master/LICENSE.md

Reply:
Fixed